packages/web-components/src/components/content-group/content-group-heading.ts

```diff
@@ -30,17 +30,11 @@ class C4DContentGroupHeading extends StableSelectorMixin(LitElement) {
   slot = 'heading';
 
   connectedCallback() {
-    if (!this.hasAttribute('role')) {
-      this.setAttribute('role', 'heading');
-    }
-    if (!this.hasAttribute('aria-level')) {
-      this.setAttribute('aria-level', '3');
-    }
     super.connectedCallback();
   }
 
   render() {
-    return html` <slot></slot> `;
+    return html` <h3><slot></slot></h3> `;
```

Comment:
This was handled differently in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/12074.